packages/Algernon.package/ALGCache.class/instance/putItems.at..st

```diff
@@ -1,4 +1,4 @@
 caching
 putItems: anItemCollection at: aCacheKey
```

Comment:
See comment for "getAt:"; I think this should be "at:put:" (or at least, "at:putItems:")